src/components/notification/notification.ts

```diff
@@ -202,7 +201,13 @@ export class SbbNotification extends LitElement {
         class="sbb-notification__wrapper"
         @transitionend=${(event: TransitionEvent) => this._onNotificationTransitionEnd(event)}
         @animationend=${(event: AnimationEvent) => this._onNotificationAnimationEnd(event)}
-        ${ref((el) => (this._notificationElement = el as HTMLElement))}
+        ${ref(async (el) => {
```

Comment:
Would it be possible to not use ref at all? I'ts just we know that with SSR it can cause problems. So when we use it as little as possible we will not run into problems once.